.github/workflows/compliance.yml

```diff
@@ -71,6 +71,7 @@ jobs:
       if: contains(github.event.pull_request.user.login, 'dependabot[bot]') != true
       run: |
         export ZEPHYR_BASE="$(dirname "$(pwd)")/zephyr"
+        export ZEPHYR_NRF_MODULE_DIR="$(dirname "$(pwd)")/nrf"
```

Comment:
as `nrf` is not given a module name here: https://github.com/nrfconnect/sdk-nrf/blob/main/zephyr/module.yml
then it obtains it's name from the path, hence they are in sync, but we should consider if a more generic approach is better.
I don't foresee the `nrf` folder to be moved or renamed, but in general having compliance relying on internal repo structure can make refactoring hard and give hard to understand errors.

A generic approach can be obtained by running `package_helper.cmake` from :
```
cmake -Ssamples/basic/blinky -Bbuild_compliance -DMODULES=zephyr_module -P cmake/package_helper.cmake
```
which will produce `build_compliance/zephyr_modules.txt` where each line is:
```
"<module_name>":"<module_dir>":"<cmake_dir>"
```
So the first two parts can be extracted and used for env settings.

Reply:
I tried, but eventually gave up.

The module_name has "-", which doesn't work well with env var names I think.

cmake -S$ZEPHYR_BASE/samples/basic/blinky -Bbuild_compliance -DMODULES=zephyr_module -P cmake/package_helper.cmake
awk -F \" '{ print "export ZEPHYR_" toupper($2) "_MODULE_DIR=" $4 }' build_compliance/zephyr_modules.txt > ZEPHYR_x_MODULE_DIR_env_vars.sh
source ZEPHYR_x_MODULE_DIR_env_vars.sh

ZEPHYR_x_MODULE_DIR_env_vars.sh: line 3: export: `ZEPHYR_WFA-QT-CONTROL-APP_MODULE_DIR=/home/sebo/ncs/modules/lib/wfa-qt-control-app': not a valid identifier